### ISSUE TITLE
Allow users to choose template from list

### ISSUE THREAD

**Issue by shytikov** [LilyBin/LilyBin#59]
Is it possible to have list of most common used templates?

For example:

* Treble Clef only;
* Treble Clef + Guitar Tab;
* Bass Clef only;
* Bass Clef + Bass Guitar Tab;
* Treble and Bass Clefs combined;

P.S. I trying to help you with pull requests, but I haven't succeed to start development environment on Windows... Working on it...

**Comment by trevordixon** [comment]
This is a good idea, I'd like this too.

**Comment by TimothyGu** [comment]
> P.S. I trying to help you with pull requests, but I haven't succeed to make development environment work on Windows... But I still keep trying.

Yeah, the current code won't work locally unless you have AWS set up. #61 should fix that.

**Comment by shytikov** [comment]
So far my problems are even more severe :) `node-pre-gyp` don't want to compile, and `docker` not able to start, since virtualbox is at daggers drawn with Hyper-V manager I use for other projects...

Trying to separate interface part and run it...

**Comment by TimothyGu** [comment]
That's my point. We are not using Docker any more but we didn't update the
README yet.

**Comment by shytikov** [comment]
Ok. So should I wait a bit to codebase will be cleaned up, or I should try to do it myself and propose pull request?

**Comment by TimothyGu** [comment]
Actually you don't need to do either. Because you will only be modifying the frontend, you don't need to worry about the backend at all. Beware that because of this you'll be seeing an error and no score on the right, but it shouldn't matter.

**Comment by PaulMorris** [comment]
I'd like to work on this.  Would probably use and/or adapt the templates from the LilyPond manuals: http://lilypond.org/doc/v2.18/Documentation/learning/templates
